libs/state/src/lib/core/accumulation-observable.ts

```diff
@@ -52,7 +52,10 @@ export function createAccumulationObservable<T extends object>(
       (state, [slice, stateAccumulator]) => stateAccumulator(state, slice),
       {} as T
     ),
-    tap(newState => (compositionObservable.state = newState)),
+    tap(
+      newState => (compositionObservable.state = newState),
+      error => console.error(error)
+    ),
```

Comment:
```suggestion
    ),
    catchError(e => EMPTY)
```

Let's catch it here to forcefully not get lost...